contents/handbook/company/team.md

```diff
@@ -502,6 +502,36 @@ When he's not working on awesome PostHog marketing, Sean can be found building a
 
 </div>
 
+<!--- Kunal Bio -->
+
+<div class="team-row" markdown="1">
+
+<div class="team-left-text" markdown="1">
+
+### Kunal Pathak, Growth Engineer
+
+<div class="team-left-bio" markdown="1">
+
+A Bay Area native and a startup vet, Kunal loves finding new ways to leverage data, product, and engineering to drive business outcomes. Most recently, he was leading the growth team at Amplitude.
```

Comment:
A small thing, all other bios are written in first person.

Reply:
oops was going off of @seanpackham 's!